tests/monotouch-test/Network/NWParametersTest.cs

```diff
@@ -199,6 +199,21 @@ public void CreateSecureTcpTestDoNotSetUpTls ()
 			}
 		}
 
+#if MONOMAC
+		[Test]
+		public void CreateCustomIP ()
+		{
+			TestRuntime.AssertXcodeVersion (11, 0);
+			byte ipVersion = 10;
+			var setUpProtocol = CreateConfigureProtocolHandler ();
+			using (var parameters = NWParameters.CreateCustomIP (ipVersion, setUpProtocol))
+			using (var endpoint = NWEndpoint.Create ("wwww.google.com", "80")) {
```

Comment:
Are the 4 `w`'s intentional?

Reply:
`:wq!` 🤣

Reply:
the api does not care, will be need to change once we merge with master to use the respires.